webhooks/constants.go

```diff
@@ -10,6 +10,7 @@ const (
 	MinTokenExprationDefault      = time.Duration(1) * time.Hour
 	DefaultGCloudRegionDefault    = "asia-northeast1"
 	GcloudImageDefault            = "google/cloud-sdk:slim"
+	VolumeModeDefault             = 440
```

Comment:
This should be `0440` (octal literal)

ref: https://go.dev/ref/spec#Integer_literals

Reply:
```suggestion
	VolumeModeDefault             = 0440
```

Reply:
When I specify it as octal, it's translated into decimal, using 288 in the generated manifest

Reply:
Yes. That is correct (288 is correct in this case). See https://kubernetes.io/docs/concepts/configuration/secret/#secret-files-permissions

Reply:
lol, I didn't know that. Thanks for the clarification 🙇